### PR TITLE
AP-4643: Add sca client relationship task

### PR DESCRIPTION
**Pull request by colinbruce**
## What

[Link to story](https://dsdmoj.atlassian.net/browse/AP-4643)

This adds the first of the relationship SCA questions.
It implements the new question on the merits task list
It handles a positive response by recording it and moving to the next task on the list, if the answer is negative, for now, it will route back to the task list.  This will need updating to move to question 2 as part of the next ticket

## Checklist

Before you ask people to review this PR:

- Tests and rubocop should be passing: `bundle exec rake`
- Github should not be reporting conflicts; you should have recently run `git rebase main`.
- The standards in the [Git Workflow document on Confluence](https://dsdmoj.atlassian.net/wiki/spaces/ATPPB/pages/4602855954/Git+Workflow) should be followed
- There should be no unnecessary whitespace changes. These make diffs harder to read and conflicts more likely.
- The PR description should say what you changed and why, with a link to the JIRA story.
- You should have looked at the diff against main and ensured that nothing unexpected is included in your changes.
- You should have checked that the commit messages say why the change was made.

### CODE REVIEW

File: app/views/providers/proceeding_merits_task/is_client_biological_parent/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+         [:biological, t("generic.yes")],
+         ["", t("generic.no")],
+       ] %>
+
+    <%= form.govuk_collection_radio_buttons :relationship_to_child,
+                                            options,
+                                            :first,
+                                            :second,
+                                            hint: { text: "" },
+                                            legend: { text: page_title, tag: "h1", size: "l" },
```
Comment (agoldstone93) [inline app/views/providers/proceeding_merits_task/is_client_biological_parent/show.html.erb:16]: Should this be xl?
```suggestion
                                            legend: { text: page_title, tag: "h1", size: "xl" },
```

**Review by agoldstone93 — APPROVED** [review]
One small question but otherwise looks good

File: spec/requests/providers/proceeding_merits_task/is_client_biological_parent_controller_spec.rb [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+          it "does not set the task to complete" do
+            patch_request
+            expect(legal_aid_application.legal_framework_merits_task_list).to have_not_started_task(:PB003, :client_relationship_to_proceeding)
+          end
+
+          it "redirects to next page" do
+            patch_request
+            expect(response).to have_http_status(:redirect)
+          end
```
Comment (agoldstone93) [inline spec/requests/providers/proceeding_merits_task/is_client_biological_parent_controller_spec.rb:79]: Should this be a specific redirect to, given that it doesn't use the steps?

**Review by agoldstone93 — APPROVED** [review]
Not sure if this has worked properly because i wanted to add another comment

Just a couple of small suggestions

File: spec/requests/providers/proceeding_merits_task/is_client_biological_parent_controller_spec.rb (hunk 2 above)
Comment (agoldstone93) [reply spec/requests/providers/proceeding_merits_task/is_client_biological_parent_controller_spec.rb:79]: I'm happy to make the change on my branch

File: app/controllers/providers/proceeding_merits_task/is_client_biological_parent_controller.rb [hunk 3]
```diff
@@ -8,7 +8,7 @@ def show
       def update
         @form = Providers::ProceedingMeritsTask::BiologicalParentForm.new(form_params)
         # TODO: change the following to redirect to question 2 when it is added
-        return redirect_to providers_legal_aid_application_merits_task_list_path(legal_aid_application) unless @form.relationship_to_child.eql?("biological")
+        return redirect_to providers_legal_aid_application_merits_task_list_path(legal_aid_application) unless @form.relationship_to_child.eql?("biological") || draft_selected?
```
[the remark this answers is not in the document]
Comment (colinbruce) [reply app/controllers/providers/proceeding_merits_task/is_client_biological_parent_controller.rb]: Discussed and updated for clarity :+1:

**Pull request merged**